module/authentication/src/Application/Security/Provider/IdUserProvider.php

```diff
@@ -57,7 +57,7 @@ public function loadUserByUsername($username)
     /**
      * {@inheritdoc}
      */
-    public function refreshUser(UserInterface $user): User
+    public function refreshUser(UserInterface $user): UserInterface
```

Comment:
Perhaps the exect instance? We know what it returns 👍🏻

Reply:
Return value of other method inside this one  is UserIntreface, i change that one to, and now is should be ok.